chaoslib/openebs/cstor_pool_health_check.yml

```diff
@@ -0,0 +1,61 @@
+- name: Wait (soak) for I/O on pools 
+  wait_for:
+    timeout: "{{ post_chaos_soak_time }}"
+
+- name: Derive PV from application PVC
+  shell: >
+    kubectl get pvc {{ app_pvc }}
+    -o custom-columns=:spec.volumeName -n {{ app_ns }}
+    --no-headers
+  args:
+    executable: /bin/bash
+  register: pv
+
+- name: Get list of pool deployments from cvr
+  shell: >
+    kubectl get cvr -n openebs
+    -l openebs.io/persistent-volume={{ pv.stdout }} --no-headers
+    -o=jsonpath='{range .items[*]}{.metadata.labels.cstorpool\.openebs\.io\/name}{"\n"}{end}'
+    | awk '{print $1}'
+  args:
+    executable: /bin/bash
+  register: pool_deployment_list
+
+- name: Get list of pool pods of respective deployments
+  shell: >
+    kubectl get pods -n openebs
+    | grep {{ item }} | grep -w "Running" | awk '{print $1}'
+  args:
+    executable: /bin/bash
+  register: pool_pod_list
+  with_items: "{{ pool_deployment_list.stdout_lines }}"
+
+- set_fact:
+    pool_pod_length: "{{ pool_pod_list.results | length }}"
+    pool_pod_named_list: "{{ pool_pod_named_list|default([])}}"
+    error_messages: 'uncorrectable I/O failure|suspended|ERROR ZFS event'
+
+- set_fact: 
+    pool_pod_named_list: "{{ pool_pod_named_list + pool_pod_list.results[item|int].stdout_lines }}"
```

Comment:
to get the list of pool pods, can we use the code in cstor_replica_network_delay.yaml?
'openebs' namespace is hardecoded in few places.. can that be parameterized?

Reply:
My bad, thanks @vishnuitta ! Pushed the test code directly.. will change it